### DISCUSSION TITLE
Questions about .mkv format video support

### DISCUSSION THREAD

**Discussion by luckyaigit** [nagadomi/nunif#177] [Q&A]
When you need to convert .mkv video, the warning "Unrecognized file type" often appears. Anyone else having the same problem?
![222](https://github.com/user-attachments/assets/fa0610e6-b385-4382-9163-18bdd5c00db4)

**Comment by nagadomi** [comment]
The reason for that error is that the mimetype for .mkv is not defined in your OS.
It is defined in my test Windows env, but there may be cases where it is not.

I changed to make .mkv explicitly defined. Run `update.bat` and try again.

**Comment by luckyaigit** [comment]
Awesome, thanks a lot for the fix!
